CHANGELOG.md

```diff
@@ -1,3 +1,81 @@
+## Fleet 4.65.0 (Mar 14, 2025)
+
+## Security engineering
+- Added UI for viewing certificate details on the host details and my device pages.
+- Added new features to include certificates in host vitals for macOS, iOS, and iPadOS.
+- Added the list host certificates (and list device's certificates) endpoints.
+- Improved the copy for the delete and transfer host modal to be more clear about the disk encryption key behavior.
+- Permit setting SSO metadata and metadata_url in gitops and UI.
+- Fixed an issue where the Show Query modal would truncate large queries.
+
+## Vulnerability management
+- Fixed Python for Windows software version mutation to avoid panics on software ingestion in some cases.
+- Prevented an invalid `FLEET_VULNERABILITIES_MAX_CONCURRENCY` value from causing deadlocks during vulnerability processing.
+- Updated default for vulnerabilities max concurrency from 5 to 1.
+- Updated CPE generation to more closely align with CPEs use in vulnerability feeds.
+- Changed software version CVE resolved in version parsing and comparison to use custom code rather than semver.
+- Added new (as of 2025-03-07) archives page to data source for MS Mac Office vulnerability feed (applies to vulnerabilities feed rather than a specific Fleet release).
+- Fixed an issue with Fleet's processing of Python versions to ensure that the correct CPEs are checked for vulnerabilities.
+- Fixed an issue with increased resource usage during vulnerabilities processing by adding database indexes.
+- Fixed false-positives on released PowerShell versions for CVE-2025-21171 and all PowerShell versions on CVE-2023-48795.
+
+## IT engineering
+- Implemented GitOps mode that locks settings in the UI that are managed by GitOps.
+- Allowed VPP apps to be automatically installed via a Fleet-created policy. 
+- Added ability for users to automatically install App Store Apps without writing a policy in the Fleet UI.
+- Updated the UI for adding and editing software for a cleaner, cohesive experience.
+- Added auto-install to FMA via the API, replacing a more brittle client-side implementation.
+- Added pagination inside each of the Manage Automations modals for policies.
+- Added UI to turn on and off Android MDM.
```

Comment:
Should we call out this is behind a feature flag? or just remove it from the changelog?

Reply:
Yeah probably a good idea, I'll do that.

Reply:
Done.